README.md

```diff
@@ -55,34 +60,6 @@ targetCrdFilters:
 The autodiscovery mechanism will create a list of all CRD names in the cluster whose names have the suffix "group1.tnf.com" or "anydomain.com", e.g. "crd1.group1.tnf.com" or "mycrd.mygroup.anydomain.com".
 
 ### testTarget
-#### podsUnderTest / containersUnderTest
-This section is usually not required if labels defined in the section above cover all resources that should be tested. If label based discovery is not sufficient, this section can be manually populated as shown in the commented part of the [sample config](test-network-function/tnf_config.yml). However, intrusive tests need to be skipped ([see here](#disable-intrusive-tests)) for a reliable test result. The pods and containers explicitly configured here are added to the target pod/container lists populated through label matching.
-
-For both configured and discovered pods/containers, the autodiscovery mechanism will attempt to identify the default network device and all the IP addresses of the pods it
-needs for network connectivity tests, though that information can be explicitly set using annotations if needed. For Pod IPs:
-
-* The annotation `test-network-function.com/multusips` is the highest priority, and must contain a JSON-encoded list of
```

Comment:
these annotations are still valid i think

Reply:
restored the section while removing the non-autodiscovery lines.